### PR TITLE
Fix #129: Unify dependency versions with Spring Boot

### LINKED ISSUE #129: Unify dependency versions with Spring Boot
Use Spring Boot BoM.

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,14 +55,21 @@
 
 		<!-- Dependencies -->
 		<spring-boot.version>2.6.8</spring-boot.version>
-		<jackson.version>2.13.3</jackson.version>
```
Comment (banterCZ) [inline pom.xml:58]: The only version to override is `h2`, otherwise we stick with Spring Boot.

File: audit-base/pom.xml [hunk 2]
```diff
@@ -55,7 +50,7 @@
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
-                <version>2.5.0</version>
+                <version>${spring-boot.version}</version>
```
Comment (banterCZ) [inline audit-base/pom.xml:53]: Unfortunately there is no support for plugin BoM https://issues.apache.org/jira/browse/MNG-5588
So at least this improvement.

**Review by petrdvorak — APPROVED** [review]
Looks OK

**Review by romanstrobl — APPROVED** [review]
Alright, I guess this is improvement.

**Pull request merged**